Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine:latest AS phrase-cli
+
+# do stuff
```

Comment:
I guess this comment is not really necessary :)

Reply:
Sorry. My bad. Keep in mind those changes should go to https://github.com/phrase/openapi/tree/master/clients/cli ;) This repo is autogenerated after each push to phrase/openapi definitions.

Reply:
@rlisowski Can you please add some more details? I am a bit confused which changes you mean and what exactly is autogenerated.

Reply:
@egze the https://github.com/phrase/openapi repo is a source of truth, if you change anything there all API clients and the CLI are re-generated and pushed to own repositories, so the changes from this PR should be in fact done in https://github.com/phrase/openapi/tree/master/clients/cli directory in phrase/openapi repo.

Reply:
So, just `1:1` copy of all changes here to the other repo?

Reply:
Not exactly 1:1, you need to copy it to clients/cli directory but yes, in general it needs to be in phrase/openapi repo.